### DISCUSSION TITLE
Dendrite Solidification code for binary alloy

### DISCUSSION THREAD

**Discussion by arpawar** [idaholab/moose#17689] [Q&A Modules: Phase field] [answered]
I am  interested in solving dendrite solidification problem for binary alloys.  I am having trouble implementing code for dendritic growth of binary alloys. Can you please help me out with some reference code for the same? Any resources will be of great help. Thank you very much!

**Comment by jiangwen84** [comment, marked as answer]
@SudiptaBiswas Could you help with this?

**Reply by SudiptaBiswas** [reply]
You can find some of the solidification examples here, 
https://github.com/idaholab/moose/tree/next/modules/phase_field/examples/anisotropic_interfaces 

The examples that use the grand-potential-based model will be relevant for binary alloys.

**Reply by KangChenRui** [reply]
> You can find some of the solidification examples here, https://github.com/idaholab/moose/tree/next/modules/phase_field/examples/anisotropic_interfaces
> 
> The examples that use the grand-potential-based model will be relevant for binary alloys.

Hi，friends

I have a question.
How to use two order parameters with the kenerl ACInterfaceKobayashi1,ACInterfaceKobayashi2 and InterfaceOrientationMaterial. 
Like op = eta1, op = eta2
